actions/lib/formatters.py

```diff
@@ -34,6 +34,7 @@ def to_issue_dict(issue, include_comments=False, include_attachments=False,
         'summary': issue.fields.summary,
         'description': issue.fields.description,
         'status': issue.fields.status.name,
+        'priority': issue.fields.priority,
```

Comment:
Can I suggest using `issue.fields.priority.name` here instead of just `issue.fields.priority`? If we leave this as is the response ends up looking something like this in the response object:

`'priority': <JIRA Priority: name='Medium', id='3'>,`

This makes it kinda hard to parse things since we are basically returning a `jira.resource.Priority` type, but I suspect what you are probably after mostly is the human readable "name" for the priority (in my example "Medium"). Does that work for you or break your particular use case?

Reply:
My use case works with either `name`, `id` or I can work with the whole object, no preference.
I left it as type in the PR as it seems `name` is a customisable fields and it might change, so some use cases might prefer to use the `id` field instead.
I agree with you though, returning the name field makes it more user friendly and ease the implementation, so I would go with that.
Should I change it in the PR?

Reply:
I'm thinking for other people consuming this pack it would be best to use `issue.fields.priority.name`. This is also consistent with what we have done with other fields (`status`, for example).

You can update this in your [branch](https://github.com/stfn775/stackstorm-jira/tree/patch-1) directly and it will be reflected in the PR. Once that is done I can +1 the PR and get it merged. Thanks!

Reply:
Done. Thanks for taking the time to review!